src/classes/RD2_Donor.cls

```diff
@@ -59,6 +60,12 @@ public with sharing class RD2_Donor {
             this.recordId = recordId;
             this.name = name;
         }
+
+        public Record(Id recordId, String name, id accId) {
```

Comment:
No need to create a new constructor, the previous one can be extended.
`id` -> `Id`